Sources/SwiftPlot/PlotStyleHelpers.swift

```diff
@@ -43,15 +43,23 @@ public struct PlotLegend {
     public init() {}
 }
 
+public struct Coordinate {
+    public enum CoordinateSpace {
+        case ndc
+    }
+    public var coordinateSpace: CoordinateSpace = .ndc
+    public var point: Point = Point(0.0, 0.0)
+}
+
 public protocol Annotation {
-    mutating func draw(renderer: Renderer)
+    mutating func draw(results: Results, renderer: Renderer)
```

Comment:
This should be:

```swift
mutating func draw(resolver: CoordinateResolver, renderer: Renderer)
```

That's why the compiler is complaining about undeclared types.

Reply:
Whoops! Silly me, thanks for catching that.